test/CMakeLists.txt

```
@@ -28,6 +37,11 @@ if (MSVC OR MSYS OR MINGW OR WIN32)
 option(BUILD_SHARED_LIBS "Build shared libraries (DLLs)." ON)
 endif()
 
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

```suggestion
if(MSVC)
```
Please use msvc instead of matching windows since this will also affect mingw and other compilers that does not accept `/*` style flags

---

Done.